File: proj/httpcachez-java/src/test/java/com/murerz/httpcachez/HttpServerTest.java
package com.murerz.httpcachez;

import static org.junit.Assert.assertTrue;

import org.junit.Test;

public class HttpServerTest {

	@Test
	public void testHttpServer() {
		HttpServer server = HttpServer.me();
		try {
			int port = server.start(0).getPort();
			assertTrue(port > 0);
		} finally {
			Util.close(server);
		}
	}

}

File: proj/darkproxy/test/HttpServerTest.js
const suite = require('./TestCase').suite;
const TestCase = require('./TestCase').TestCase;
const assert = require('assert');
const HttpServer = require('../src/http/HttpServer').HttpServer;
const HttpClient = require('../src/http/HttpServer').HttpClient;

class HttpServerTest extends TestCase {

  async testServer() {
    //let server = new HttpServer({port: 0});
    //await server.start();
    throw 2;
  }


}

suite.add(new HttpServerTest());
